lib/setup.py

```diff
@@ -30,7 +30,7 @@
 # - Include relevant lower bound for any features we use from our dependencies
 # - And include an upper bound that's < NEXT_MAJOR_VERSION
 INSTALL_REQUIRES = [
-    "altair>=3.2.0, <5",
```

Comment:
Is this breaking for Altair 3? We had conversations about 4 to 5, but want to verify the problem with 3.

Reply:
Ah Lukas pointed this out, but we already used v4 as well, so we were probably already breaking.

Reply:
The [3.x version](https://github.com/altair-viz/altair/tree/v3.3.0/altair/vegalite) does not even contain the `v4` import we are using, so I'm pretty sure that we haven't been supported this version anyways for a while.